ServantHDBC/src/Api/Photo.hs

```diff
@@ -0,0 +1,25 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeOperators #-}
+
+module Api.Photo (PhotoAPI, photoHandlers) where
+
+import Control.Monad.Trans.Except
+import Servant
+
+type PhotoAPI = "photos" :>
+                (Capture "path_segment_1" String :>
```

Comment:
How is `Capture` different from `Query`? What does `path_segment_1` map to in the incoming request?